interactions/api/models/channel.pyi

```diff
@@ -1,14 +1,25 @@
 from datetime import datetime
 from enum import IntEnum
-from typing import List, Optional, Union
+from typing import List, Optional, Union, Callable
 
 from .message import Message, Embed, MessageInteraction
 from ...models.component import ActionRow, Button, SelectMenu
 from .misc import DictSerializerMixin, Overwrite, Snowflake, MISSING
 from .user import User
 from ..http import HTTPClient
 
-class ChannelType(IntEnum): ...
+class ChannelType(IntEnum):
```

Comment:
Writing `IntEnum` metaclasses should be handled in the regular `.py` file, this is for interfacing. For `.pyi` usage, I recommend continuing to use the ellipsis type.